### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by hansven**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## First steps
+Thank you for getting involved! Before you start, please make sure to familiarize yourself with the documentation on https://www.marblerun.sh/docs/introduction/
+
+Please take note of our [Code of Conduct](CODE_OF_CONDUCT.md) and follow it when interacting with our project.
```
Comment (thomasten) [inline CONTRIBUTING.md:4]:
```suggestion
Thank you for getting involved! Before you start, please familiarize yourself with the [documentation](https://www.marblerun.sh/docs/introduction/).

Please follow our [Code of Conduct](CODE_OF_CONDUCT.md) when interacting with this project.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+If you want to support our development:
+* Add a GitHub Star to the project
+* Share our projects on social media
+* Join the [Confidential Computing Discord](https://discord.gg/rH8QTH56JN)
+
+Marblerun is licensed under the [Mozilla Public License 2.0](https://github.com/edgelesssys/marblerun/blob/master/LICENSE). When contributing, you also need to agree to our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/edgelessrt).
```
Comment (thomasten) [inline CONTRIBUTING.md:11]:
```suggestion
Marblerun is licensed under the [Mozilla Public License 2.0](LICENSE). When contributing, you also need to agree to our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/marblerun).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Development guidelines
+
+
+Use the standard [Go styleguide](https://golang.org/doc/effective_go.html). Read [Common Review Comments](https://github.com/golang/go/wiki/CodeReviewComments) for further information.
```
Comment (thomasten) [inline CONTRIBUTING.md:16]:
```suggestion
Adhere to the style and best practices described in [Effective Go](https://golang.org/doc/effective_go.html). Read [Common Review Comments](https://github.com/golang/go/wiki/CodeReviewComments) for further information.
```
Effective Go is not about style only

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Pull request process
+
+Submissions should remain focused in scope and avoid containing unrelated commits.
+For pull requests, we employ the following workflow:
+1. Fork the repository to your own GitHub account
+2. Create a branch locally with a descriptive name
+3. Commit changes to the branch
+4. Write your code according to our development philosophy
```
Comment (thomasten) [inline CONTRIBUTING.md:26]:
```suggestion
4. Write your code according to our development guidelines
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+To report a security issue, contact security@edgeless.systems.
+
+Your bug report should cover the following points:
+*	A quick summary and/or background of the issue
+*	Steps to reproduce (be specific, e.g., provide sample code)
+*	What you expected would happen
+*	What actually happens
+*	Further notes:
+    * Toughts on possible causes
```
Comment (thomasten) [inline CONTRIBUTING.md:43]:
```suggestion
    * Thoughts on possible causes
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    * Tested workarounds or fixes
+
+## Major changes and feature requests
+Larger changes to the codebase are best discussed with its maintainers before starting – you can get in touch at contact@edgeless.systems. For more immediate communication with our Devs and other community members please join our Discord server https://discord.com/invite/rH8QTH56JN
```
Comment (thomasten) [inline CONTRIBUTING.md:47]: remember to replace this paragraph with the one from ERT

**Review by thomasten — CHANGES_REQUESTED** [review]

**Comment by 3u13r** [comment]
Also added all applicable change requests to the ert branch.

**Review by thomasten — APPROVED** [review]

**Pull request merged**